bitcoin/src/psbt/mod.rs

```diff
@@ -44,15 +44,54 @@ use self::map::Map;
 /// Partially signed transaction, commonly referred to as a PSBT.
 pub type Psbt = PartiallySignedTransaction;
 
+/// Future version of PSBT this library can't parse
+#[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug)]
```

Comment:
I'm not sure if we should keep `Ord` and `PartialOrd`. We can probably keep the others.